src/components/CartItems/CartItem/CartItem.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { ReactComponent as DecreaseQntCounter } from '../../../assets/icons/cartIcons/decrease.svg';
+import { ReactComponent as IncreaseQntIcon } from '../../../assets/icons/cartIcons/increase.svg';
+import { ReactComponent as RemoveItemIcon } from '../../../assets/icons/cartIcons/remove.svg';
+import {
+  fetchPoductUpdate,
+  fetchProductRemoveFromCart,
+} from '../../../axios/cart/requests';
+import { getPriceToFixed } from '../../../helpers/functions';
+import { showAlert } from '../../../store/actions/alert';
+import {
+  removeProductFromCart,
+  updateProductInCart,
+} from '../../../store/actions/cart';
+import Button from '../../UI/Button/Button';
+import QuantityLoader from '../../UI/QuantityLoader/QuantityLoader';
+import classes from './CartItem.module.css';
+
+// displaying separate cart items
+
+const CartItem = ({ item }) => {
+  const { id, name, price, pictureUrl, quantity } = item;
+  const modPrice = getPriceToFixed(price, 2);
+  const [qantityLoader, setQuantityLoader] = useState(false);
+  const dispatch = useDispatch();
+
+  const onRemoveHandler = async () => {
+    // eslint-disable-next-line no-restricted-globals
+    if (!confirm(`Are you sure to delete ${name} from your order?`)) return;
+
+    const response = await fetchProductRemoveFromCart(id);
+
+    if (!response.isError) {
```

Comment:
This is the place to catch an exception!
Wrap `await fetchProductRemoveFromCart(id);` with try / catch instead of if / else.

Reply:
you are right, changes are here https://github.com/maxSedelnikov/react-shopping-app/pull/1/commits/24984c3c8b52e6eb5707ea4a4edfa8373dc5a402